doc/source/extended_documentation/calibration/rainforests_calibration/rainforests_calibration.rst

```diff
@@ -0,0 +1,173 @@
+#######################################
+RainForests calibration
+#######################################
+
+RainForests calibration is a situation dependent non-parametric method for the calibration
+of rainfall based on the ECPoint method of Hewson and Pillosu `Hewson & Pillosu, 2021`_.
+
+.. _Hewson & Pillosu, 2021: https://www.nature.com/articles/s43247-021-00185-9
+
+****************************************************
+Sub-grid variability as a means of calibration
+****************************************************
+
+ECPoint is based on the principle of using sub-grid variability as a means to calibrate
+grid-scale rainfall forecasts. Sub-grid variability in this context is the connection
+between the point observations one would measure within a grid-box when presented with
+a given areal average grid-scale forecast value. This relationship is encapsulated in the
+mapping that takes a grid-scale forecast value and maps it onto a distribution of typical
+point observation values, or equivalently a set of forecast error values, and is central
+to this calibration method.
+
+Naturally the relationship between this distribution and the forecast value is contingent
+on the underlying rainfall processes at play. For instance, the distribution associated
+with a rainband will be characteristically different to that associated with post-frontal
+showers and different still to that associated with deep-tropical convection. To this end,
+a suitable set of meteorological variables can be used to distinguish different rainfall
+regimes and identify the associated distribution that describes the underlying sub-grid
+variability.
+
+Equipped with the relevant distribution that describes the sub-grid variability for a given
+forecast, the grid-scale forecast value can be mapped onto a series of realisable point-scale
+values, yielding a form of conditional bias-correction. In the context of ensemble forecasts,
+this mapping is undertaken on a per realization basis to produce a series of realisable
+point-scale values which collectively form a calibrated ensemble, referred herein as a
+"super-ensemble".
+
+Hewson & Pillosu propose a framework (ECPoint) to determine and apply these distributions
+across a variety of weather types through the use of a manually tuned decision tree model
```

Comment:
Maybe avoid using "tuned" here, since it's often used in the context of ML to mean adjusting model hyper-parameters, whereas here we mean actually building the decision tree itself. I think it could be replaced by "manually constructed", "manually designed" or similar.

Reply:
Good suggestion. I think I like "manually constructed" better, so I'll go with that.